plugin/src/test/java/org/opensearch/ml/action/models/SearchModelTransportActionTests.java

```diff
@@ -5,20 +5,28 @@
 
 package org.opensearch.ml.action.models;
 
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.*;
```

Comment:
avoid using import *?

Reply:
Thanks for the remind. I have changed my intellij to not do this again.